### ISSUE TITLE
[uq-uncertainty] vanilla accuracy 20 news (even classes) unreachable

### ISSUE THREAD

**Issue by Jordy-VL** [google-research/google-research#393]
The appendix of the paper mentions reaching 95.5% accuracy for 20news with the detailed hyperparameter settings. 
Whatever I do, I can never exceed 40% accuracy on the in-distribution data [even data]. 
Did they mean AUC? 

I add in my Sacred config for comparison: 
```@ex.named_config
def ovadia_20news():
    """
    %maxdoclen 250; max_vocabulary 30000
    %The vanilla model uses a one-layer LSTM model of size 32 and a dense layer to predict the 10 class
    % probabilities based on word embedding of size 128. A dropout rate of 0.1 is applied to both the LSTM
    % layer and the dense layer for the Dropout model. The LL-SVI model replaces the last dense layer
    % with a Bayesian layer, the ensemble model aggregates 10 vanilla models, and stochastic methods
    % sample 5 predictions per example. The vanilla model accuracy for in-distribution test data is 0.955.
    """
    identifier = "20news"
    data_folder = os.path.join(DATAROOT, identifier)
    out_folder = generate_out_folder(data_folder)
    task = "document_classification"  # or regression
    max_vocabulary = 30000  # 20news to 30K e.g.
    composition = ["word"]

    model = "lstm"
    embed_dim = 128
    projection_nodes = 32
    dropout = 0.1
    dropout_nonlinear = 0.1
    embedding_dropout = 0
    dropout_concrete = None
    weight_decay = 0  # .0001  # triggers AdamW optimizer
    max_document_len = 250

    epochs = 48
    optimizer = "adam"
    clipnorm = 10
    learning_rate = 0.001
    steps_per_epoch = None  # could also be None

    posterior_sampling = 10
    use_aleatorics = False
    multilabel = False
    loss_fn = "categorical_crossentropy" if not use_aleatorics else "attenuated_learned_loss"
    metrics = ["accuracy", "mse"] if not use_aleatorics else []
    ood = ['comp.graphics', 'comp.sys.ibm.pc.hardware', 'comp.windows.x', 'rec.autos', 'rec.sport.baseball', 'sci.crypt', 'sci.med', 'soc.religion.christian', 'talk.politics.mideast', 'talk.religion.misc']
```

**Comment by Jordy-VL** [comment]
@JasperSnoek for example

**Comment by jessieren** [comment]
Hi Jordy-VL,

May I ask how you processed the raw dataset? Did you use the data_generate.py script?

Thanks.

Jie

**Comment by Jordy-VL** [comment]
I used the same Hendrycks 20news data; yet I used a separate script to read
the data.
